apps/site/assets/ts/stop/__tests__/DeparturesAndMapTest.tsx

```diff
@@ -48,12 +48,6 @@ beforeEach(() => {
     .spyOn(useSchedules, "useSchedulesByStop")
     .mockReturnValue({ status: FetchStatus.Data, data: schedules });
   jest.spyOn(useVehiclesChannel, "default").mockReturnValue([]);
-  jest.mock("react-router-dom", () => ({
-    __esModule: true,
-    useLoaderData: () => {
-      return TEST_LOADER_VALUE;
-    }
-  }));
```

Comment:
note: This was redundant because the data is already passed into the render function inside the helper function `renderWithRouter`